Tasks/DownloadPackageV1/multifilepackage.ts

```diff
@@ -49,7 +49,8 @@ export class MultiFilePackage extends Package {
     private async getPackageFileContent(fileMetadata: any): Promise<Map<string, PackageFileResult>> {
         return new Promise<Map<string, PackageFileResult>>((resolve, reject) => {
             var resultMap = new Map<string, PackageFileResult>();
-            var content = fileMetadata.protocolMetadata.data.content as string;
+            let fileContent = fileMetadata.protocolMetadata.data.content || fileMetadata.protocolMetadata.data.Content;
+            var content = fileContent as string;
```

Comment:
nit: `var` -> `let`

Reply:
If you're going to change it maybe do it for all other occurrences in this fn\file